src/boot/store.js

```diff
@@ -217,6 +217,9 @@ const migrations: { [string]: (GlobalState) => GlobalState } = {
     narrows: Immutable.Map(state.narrows),
   }),
 
+  // Change format of keys representing narrows.
+  '17': dropCache,
```

Comment:
Ah, good catch!

I think I'll add a real migration, then. It's probably not real common for someone to happen to have a draft they've just written (or otherwise written recently enough to still want it) at the time the app gets upgraded -- it's possible the number of users who'd notice, on a given upgrade, would be zero -- but if they do it'd be best not to discard it.